docs/pages/en-us/overview.md

```diff
@@ -15,37 +15,22 @@ banner:
 
 ## User and System Actions
-Applications define user and system actions in an audit event definition file. This definition is registered using the Audit Management web service API to create the Vertica database schema for storing the audit event messages. The definition is also utilized to generate a Java SDK, which raises the defined audit events.
+Applications define user and system actions in an audit event definition file.
 
 For more details on the audit events definition schema, see [Getting Started](Getting-Started).
 
-## Java SDK
-The audit events definition file is used by a code generation plugin to auto-generate a Java library, which sends audit event messages to the Apache Kafka messaging service.
+## Audit Library
+The audit events definition file is used by a code generation plugin to auto-generate a client-side Java library, which provide type safety and sends audit event messages to Elasticsearch.
 
-For more details on the Auditing Java SDK, see [Client-API](Client-API).
+For more details on the Java Client-Side Audit Library, see [Client-API](Client-API).
 
-## Apache Kafka
-Apache Kafka is a distributed, partitioned, replicated commit log service that provides the functionality of a messaging system. The Auditing Java SDK sends user and system audit events to Apache Kafka. These event messages are then streamed from Apache Kafka into the Vertica database schema.
-
-For more details on the Apache Kafka deployment, see [Getting Started](Getting-Started).
-
-## Vertica
-User and system audit events are stored in Vertica. The database schema for these events is created when each new tenant is registered using the Audit Management web service API. Vertica provides a high-performance loading mechanism for streaming the data from Apache Kafka into the Vertica database.
-
-For more details on the Vertica deployment, see [Getting Started](Getting-Started).
-
-## Management Web API
-The Audit Management web service API is a RESTful web service that makes it easy for you to register application audit events and new tenants.
-
-To see the web methods in the Audit Management web service API, see [Server-API](Server-API).
-
-For more details on the architecture of the Audit Management web service API, see [Architecture](Architecture).
-
-For instructions on deploying and using the Audit Management web service API, see [Getting Started](Getting-Started).
+## Elasticsearch
+Elasticsearch is an opensource, distributable, scalable, enterprise grade search engine.  Elasticsearch is accessible through an extensive RESTful API and can provide multi-tenant capable full text searches that support data discovery.  Official clients are available in Java, .NET (C#), Python, Groovy and many other languages.
 
+For more details on Elasticsearch deployment, see [Audit Service Deploy Project in GitHub](https://github.com/CAFAudit/audit-service-deploy).
```

Comment:
I would include a link to Elasticsearch's homepage too for readers that need more details on it.